website/src/components/Quiz.js

```diff
@@ -1,5 +1,6 @@
 import { Button, Card } from 'antd';
 import React, { useState } from 'react';
+import Latex from 'react-latex-next';
```

Comment:
consider to use default math equation engine 

https://docusaurus.io/docs/markdown-features/math-equations#configuration